include/vrv/object.h

```diff
@@ -1094,6 +1098,10 @@ class Object : public BoundingBox {
 
     static bool sortByUlx(Object *a, Object *b);
 
+    virtual int FindNextOfType(FunctorParams *);
```

Comment:
Same naming concern as before. Also place this declaration with other functions used by functors for organization.